bootstrap.d/dev-libs.yml

```diff
@@ -526,6 +526,40 @@ packages:
           DESTDIR: '@THIS_COLLECT_DIR@'
         quiet: true
 
+  - name: libotf
+    source:
+      git: 'https://salsa.debian.org/debian/libotf.git'
+      branch: 'master'
+      commit: '85af96e78b5ba551d98eb994f010e737ee044b94'
+      version: '0.9.16'
+      tools_required:
+        - host-autoconf-v2.69
+        - host-automake-v1.15
+        - host-libtool
+        - host-pkg-config
+      regenerate:
+        - args: ['./autogen.sh']
+        - args: ['cp',
+            '@BUILD_ROOT@/tools/host-automake-v1.15/share/automake-1.15/config.sub',
+            '@THIS_SOURCE_DIR@/']
```

Comment:
autogen doesn't regenerate this automatically?

Reply:
Nope it couldnt find managarm as the host.